### ISSUE TITLE
Problem with fresh install

### ISSUE THREAD

**Issue by abixadamj** [ltsp/community#42]
I have ZorinOS installed and fresh LTSP 19.
When I start VirtualBox Client with ethernet bridged card, ipxe starts, but after that i see an error:
![obraz](https://user-images.githubusercontent.com/5088643/67806318-e8d5b680-fa92-11e9-8da7-ea4b66f5e9ed.png)

My server config is:
```

root@ltsp-test:~# ltsp info
LTSP version: 19.10-1~201910290653~ubuntu18.04.1

SERVER XSESSIONS: zorin.desktop

SERVER OS: PRETTY_NAME="Zorin OS 15"

CHROOTS:


VMs:


IMAGES:
x86_64

root@ltsp-test:~# ltsp ipxe
Installed /usr/share/ltsp/server/ipxe/ltsp.ipxe in /srv/tftp/ltsp/ltsp.ipxe
Skipping existing /srv/tftp/ltsp/memtest.0
Skipping existing /srv/tftp/ltsp/memtest.efi
Skipping existing /srv/tftp/ltsp/snponly.efi
Skipping existing /srv/tftp/ltsp/undionly.kpxe
Installed iPXE binaries and configuration:
-rw-r--r-- 1 root root 2990 paź 29 21:28 /srv/tftp/ltsp/ltsp.ipxe
-rw-r--r-- 1 root root 161968 lip 24 12:10 /srv/tftp/ltsp/memtest.0
-rw-r--r-- 1 root root 1235864 lip 24 12:10 /srv/tftp/ltsp/memtest.efi
-rw-r--r-- 1 root root 159296 lip 24 12:10 /srv/tftp/ltsp/snponly.efi
-rw-r--r-- 1 root root 67642 lip 24 12:10 /srv/tftp/ltsp/undionly.kpxe

root@ltsp-test:~# ls -la /srv/ltsp/images/
razem 4652528
drwxr-xr-x 2 root root       4096 paź 29 20:59 .
drwxr-xr-x 3 root root       4096 paź 29 20:36 ..
-rw-r--r-- 1 root root 4764176384 paź 29 20:59 x86_64.img
root@ltsp-test:~# uname -a
Linux ltsp-test 5.0.0-32-generic ltsp/community#22~18.04.2-Ubuntu SMP Thu Oct 10 10:36:02 UTC 2019 x86_64 x86_64 x86_64 GNU/Linux
root@ltsp-test:~# 

```
VirtualBox configured as follows:
![obraz](https://user-images.githubusercontent.com/5088643/67806559-68638580-fa93-11e9-919b-eb1f321efb01.png)

![obraz](https://user-images.githubusercontent.com/5088643/67806714-b7111f80-fa93-11e9-98c2-47a5f9ad98fd.png)

The same hardware and software (another zorin on the same server) with ltsp-pnp works OK.
@alkisg - can you help me, what should I check more?

**Comment by alkisg** [comment]
I think that you have a 32bit VM and a 64bit image,
so you just need a 64bit VM.

**Comment by abixadamj** [comment]
No, I have 64bit VM
![obraz](https://user-images.githubusercontent.com/5088643/67834845-bf477a00-fae8-11e9-99c7-195b585a2515.png)

**Comment by abixadamj** [comment]
I am doing image once again:
```

ltsp@ltsp-test:~$ sudo -i
[sudo] hasło użytkownika ltsp:     
root@ltsp-test:~# ltsp image /
Using x86_64 as the base name of image /
Running: mount --bind --make-private -o ro / /tmp/tmp.HEzBqASNfZ/root/
Trying to acquire package management lock: //var/lib/dpkg/lock
Running: mount -t overlay -o upperdir=/tmp/tmp.HEzBqASNfZ/tmpfs/up,lowerdir=/tmp/tmp.HEzBqASNfZ/root,workdir=/tmp/tmp.HEzBqASNfZ/tmpfs/work /tmp/tmp.HEzBqASNfZ/tmpfs /tmp/tmp.HEzBqASNfZ/root
Cleaning up x86_64 before mksquashfs...
Replacing /tmp/tmp.HEzBqASNfZ/root/etc/ssh/ssh_host_ecdsa_key
Replacing /tmp/tmp.HEzBqASNfZ/root/etc/ssh/ssh_host_ed25519_key
Replacing /tmp/tmp.HEzBqASNfZ/root/etc/ssh/ssh_host_rsa_key
Can't load /root/.rnd into RNG
140217772069952:error:2406F079:random number generator:RAND_load_file:Cannot open file:../crypto/rand/randfile.c:88:Filename=/root/.rnd
Generating a RSA private key
.....................................+++++
.......+++++
writing new private key to '/tmp/tmp.HEzBqASNfZ/root/etc/ssl/private/ssl-cert-snakeoil.key'
-----
Parallel mksquashfs: Using 2 processors
Creating 4.0 filesystem on /srv/ltsp/images/x86_64.img.tmp, block size 131072.

```

**Comment by alkisg** [comment]
Ah ok the problem is this: https://github.com/coreos/tectonic-installer/issues/932#issuecomment-305866416

I.e. the embedded iPXE that is included with VirtualBox, doesn't support .bz images, and Zorin probably uses .bz.

So, download ipxe.iso and configure VirtualBox to boot from the CD, not from the network, so that it's using a normal iPXE.

**Comment by abixadamj** [comment]
Yes,
I have replaced VBox with real PC and it is better, but now I have:

![IMG_20191030_083952](https://user-images.githubusercontent.com/5088643/67839057-fd499b80-faf2-11e9-941a-45f7a0b3eb00.jpg)


```
Oct 30 08:31:47 ltsp-test dnsmasq-tftp[777]: sent /srv/tftp/ltsp/ltsp.ipxe to 192.168.22.167
Oct 30 08:31:53 ltsp-test dnsmasq-tftp[777]: sent /srv/tftp/ltsp/x86_64/vmlinuz to 192.168.22.167
Oct 30 08:31:53 ltsp-test dnsmasq-tftp[777]: file /srv/tftp/ltsp/ltsp.img not found

# I have:
root@ltsp-test:/srv/ltsp/images# ls -la
razem 9305212
drwxr-xr-x 2 root root       4096 paź 30 08:05 .
drwxr-xr-x 3 root root       4096 paź 29 20:36 ..
-rw-r--r-- 1 root root 4764344320 paź 30 08:05 x86_64.img
-rw-r--r-- 1 root root 4764176384 paź 29 20:59 x86_64.img.old


```
But in file **/srv/tftp/ltsp/ltsp.ipxe** 
i have - i think - good entry:

```
:start
# To completely hide the menu, set menu-timeout to -1
isset ${menu-timeout} || set menu-timeout 5000
iseq "${menu-timeout}" "-1" && goto ${img} ||
menu iPXE boot menu - ${hostname}:${srv}:${root-path} || goto ${img}
item --gap                        Boot an image from the network in LTSP mode:
item --key 1 x86_64               x86_64.img

:x86_64
:images
# The "images" method can boot anything in /srv/ltsp/images
set cmdline_method root=/dev/nfs nfsroot=${srv}:/srv/ltsp ltsp.image=images/${img}.img loop.max_part=9
goto ltsp
 
```

**Comment by alkisg** [comment]
You can still use virtualbox; just attach https://boot.ipxe.org/ipxe.iso to the virtual client.

I think now you just forgot to run the `sudo ltsp initrd` of the installation page.

**Comment by abixadamj** [comment]
So, I think you are 125% right ;-)
```

root@ltsp-test:/srv/ltsp/images# ltsp initrd
278 bloków
Generated ltsp.img:
-rw-r--r-- 1 root root 142336 paź 30 08:57 /srv/tftp/ltsp/ltsp.img

```
Now is much more better ;-) 
Now I have little problem with NFS, but it is 10 minuts to work...
```
Oct 30 09:00:07 ltsp-test dnsmasq-tftp[777]: sent /srv/tftp/ltsp/ltsp.ipxe to 192.168.22.167
Oct 30 09:00:12 ltsp-test dnsmasq-tftp[777]: sent /srv/tftp/ltsp/x86_64/vmlinuz to 192.168.22.167
Oct 30 09:00:12 ltsp-test dnsmasq-tftp[777]: sent /srv/tftp/ltsp/ltsp.img to 192.168.22.167
Oct 30 09:00:19 ltsp-test dnsmasq-tftp[777]: sent /srv/tftp/ltsp/x86_64/initrd.img to 192.168.22.167
Oct 30 09:00:29 ltsp-test dnsmasq-dhcp[777]: 465908553 available DHCP subnet: 192.168.22.0/255.255.255.0
Oct 30 09:00:29 ltsp-test dnsmasq-dhcp[777]: 465908553 vendor class: Linux ipconfig
Oct 30 09:00:29 ltsp-test dnsmasq-dhcp[777]: 465908553 available DHCP subnet: 192.168.22.0/255.255.255.0
Oct 30 09:00:29 ltsp-test dnsmasq-dhcp[777]: 465908553 vendor class: Linux ipconfig
Oct 30 09:00:29 ltsp-test rpc.mountd[744]: refused mount request from 192.168.22.167 for /srv/ltsp (/srv/ltsp): unmatched host

```

**Comment by abixadamj** [comment]
OK, after I did the last command:
`ltsp nfs`
Everything works ;-) Thank You for help.
